qml/pages/VenueMapPage.qml

```diff
@@ -48,7 +48,7 @@ BVApp.Page {
 
             onClicked: {
                 var query = venueCoordinate.latitude + "," + venueCoordinate.longitude
-                if (BVApp.Theme.isIos || BVApp.Theme.isOSX) {
+                if (BVApp.Platform.isIos || BVApp.Platform.isMacOS) {
                     Qt.openUrlExternally("https://maps.apple.com/?q=" + query)
```

Comment:
There is a possibility to choose between Apple Maps or Google Maps on iOS, but I have not found out yet how to do it. However I am not happy about the patch: v-play has Theme.isIos, Theme.isOSX, ... and I wanted it to be as close to it as possible... Could you take out that patch or revert it or something?!